backend/app/models/file.py

```
@@ -92,11 +92,22 @@ class AudioUploadResponse(SQLModel):
     content_type: str = Field(..., description="MIME type of the audio file")
 
 
+class AudioUploadResponse(SQLModel):
```

Ruff (0.15.5)

[error] Redefinition of unused `AudioUploadResponse` from line 85: `AudioUploadResponse` redefined here

(F811)